scripts/checkconfig.py

```diff
@@ -62,7 +62,7 @@
 
 def separate_location_lines(dirs):
     for dir in dirs:
-        print "    ", dir
+        print ("    ", dir)
```

Comment:
Done. Is there anyway to check coding style errors in python scripts like we have checkpatch.pl for c files ?